test/views/templates/ChangeSuccessViewSpec.scala

```diff
@@ -60,7 +60,7 @@ class ChangeSuccessViewSpec extends ViewBaseSpec {
           }
 
           "have a finish button with the correct text" in {
-            elementText(Selectors.button) shouldBe "Back to client’s details"
+            elementText(Selectors.button) shouldBe "Finish"
           }
 
```

Comment:
Following from the convo in the dev channel I think it would be best to ensure that we're testing where this link is actually handing off too (which will be BTA here as the FS appears to be on in this section). It will be a similar test as ones below it for the same thing and the btaAccountDetailsUrl already exists in MockAppConfig. Lemme know if you want a hand though